pkg/operator/configobserver/cloudprovider/observe_cloudprovider.go

```diff
@@ -188,6 +188,8 @@ func GetPlatformName(platformType configv1.PlatformType, recorder events.Recorde
 	case configv1.LibvirtPlatformType:
 	case configv1.OpenStackPlatformType:
 		cloudProvider = "openstack"
+	case configv1.IBMCloudPlatformType:
+		cloudProvider = "ibmcloud"
```

Comment:
Should we include IBM in `validCloudProviders := sets.NewString("aws", "azure", "gce", "openstack", "vsphere")` above?  It appears that we should.

Reply:
no, we don't need to include IBM there. That list defines what in-tree cloud providers require `--cloud-config` parameter. For external cloud providers we don't need it.

Reply:
At the moment, KCMO uses this name to sync the cloud-config configmap into its namespace for platforms that need it. IBMCloud doesn't need this as KCMO doesn't know anything about IBMCloud, so this case should return empty string as Ovirt and Kubevirt do below

Reply:
PR updated based on these comments.